src/app/utilities/api-clients/collections.js

```diff
@@ -30,6 +30,27 @@ export default class collections {
         return http.put(`/zebedee/collection/${collectionID}`, body);
     }
 
+    static getContent(collectionID) {
+        return http.get(`/zebedee/data/${collectionID}?uri=/`);
```

Comment:
With the param `?uri=/` isn't this only getting homepage content?

Reply:
Yeah - realised this was a method no longer being used and had forgotten to remove it. Have removed it now